### DISCUSSION TITLE
Does mellon can be supported in other platforms? Such as Windows, Solaris?

### DISCUSSION THREAD

**Discussion by hisupermodel** [latchset/mod_auth_mellon#53] [General]
Hello,  developers. (@simo5 , @thijskh ...)can you help me confirm one thing. Is there any version of mellons to run in Windows or Solaris?

Currently my system works in windows server, I want to use mellon to realize authentication. So is there good suggestion to tell me how can I do? 

Thanks.

**Comment by thijskh** [comment]
I’m not aware of anything that specifically prohibits this. And we’re received reports about Solaris so I’m quite sure that it works there. Maybe just try it?

**Comment by hisupermodel** [comment]
Thanks for your reply. @thijskh 
So the release version file (mod_auth_mellon-0.17.0.tar.gz) can run directly in Windows Server and Solaris platforms? Or is there Windows version mellon file? Or Solaris version file? How can I get them?

Currently, I run successfully in Linux system. So the config information (both Windows and Solaris) are similar to Linux?
Are there minimum dependencies on Windows/Solaris?

My apache environments are 2.0, 2.2, and 2.4.
System environments are Window server 2016, Solaris 10&11.

Thanks.

**Comment by 3eka** [comment]
On 07.12.2020 16:52, Thijs Kinkhorst wrote:
>
> I’m not aware of anything that specifically prohibits this. And we’re 
> received reports about Solaris so I’m quite sure that it works there. 
> Maybe just try it?
>
Hi all,


on Illumos, OpenIndiana we have package:
:; pkg contents web/server/apache-24/module/apache-mellon
PATH
etc/apache2/2.4/samples-conf.d/mellon.conf
usr/apache2/2.4/bin/mellon_create_metadata.sh
usr/apache2/2.4/libexec/mod_auth_mellon.so
usr/share/doc/mellon/README.md

Since this is Solaris like, it seems that it could work.


Or, if you want to compile it, check 
https://github.com/OpenIndiana/oi-userland/blob/oi/hipster/components/web/apache2-modules/mod_auth_mellon/Makefile


Regards.

**Comment by hisupermodel** [comment]
Thanks, @3eka  How about in windows server? such as 2016.
